sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.io.Serializable;
+
+/**
+ * Represent the operation user was performing when the error happened.
```

Comment:
Some of the operations may not be performed by the user. For e.g autocomplete, autorenew or even abandoning a message.